service/tablet_allocator.cc

```diff
@@ -74,7 +81,24 @@ class load_balancer_stats_manager {
                            stats.load)(dc_lb)(node_lb)
         });
     }
+
+    void setup_metrics(load_balancer_cluster_stats& stats) {
+        namespace sm = seastar::metrics;
+        // FIXME: we can probably improve it by making it per resize type (split, merge or none).
```

Comment:
These metrics will trigger very rarely (can have weeks with no change), so I don't think they need to be too fine grained.

Reply:
Ok. We also plan on providing observability with virtual tables (one for group0 leader, another for replicas)